apps/admin-x-activitypub/src/api/activitypub.test.ts

```diff
@@ -1255,8 +957,16 @@ describe('ActivityPubAPI', function () {
         });
     });
 
-    describe('getFeed', function () {
-        test('It returns an array of posts in a feed', async function () {
+    // Use the helper function to test each endpoint that returns a paginated list of posts
+    testPaginatedApiEndpoint('getFeed', '.ghost/activitypub/feed');
+    testPaginatedApiEndpoint('getInbox', '.ghost/activitypub/inbox');
+    testPaginatedApiEndpoint('getPostsByAccount', '.ghost/activitypub/posts');
+    testPaginatedApiEndpoint('getPostsLikedByAccount', '.ghost/activitypub/posts/liked');
```

Comment:
🤔 I'm not sure about this, if one of the tests fail, how will i know which endpoint it has failed for?

Reply:
I have removed the refactoring from these tests for now. I'll do the refactoring in a separate PR